Studentmediene/kapina-frontend
# Fix/kap 231 update babel react redux

BLOCKED BY https://github.com/Studentmediene/kapina-frontend/pull/245. Fix that one first.

This PR updates Babel to v7.x.x, with lots of breaking changes to previous versions. Also updates Foorumi, as the syntax used there was actually types, not values.

To test:
- Delete node_modules
- Install all dependencies with `npm install`
- Run test, `npm test`. All tests should pass.
- Try building, `npm run build`. No errors should be thrown.

If you have trouble, try upgrading NodeJS on your machine.
